c2corg_ui/static/js/documentediting.js

```diff
@@ -54,7 +55,14 @@ app.module.directive('appDocumentEditing', app.documentEditingDirective);
  * @export
  */
 app.DocumentEditingController = function($scope, $element, $attrs,
-    appAuthentication, appAlerts, appApi, authUrl) {
+    appAuthentication, appAlerts, appApi, authUrl, apiUrl, ngeoLocation) {
```

Comment:
The ordering of the parameters is also unconsistent. I am wondering why you don't get an error from closure.